src/registry/Registry.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * The registry that will have the singleton
+ * - getRegistry() -> the registry which is a simple key/value POJO
+ * @type {Object}
+ */
+const Registry = {
```

Comment:
It's a widget registry, just a hook to redefine all pure components (TextWidget, CheckboxWidget, etc.). 
I can't imagine that you would start using `react-bootstrap` for one form and `material-ui` for another one :/